src/plugins/intel_cpu/src/nodes/subgraph.h

```diff
@@ -172,10 +173,48 @@ class Subgraph::SubgraphExecutor {
     inline void segfault_detector();
 #endif
 
-private:
-    std::vector<MemoryPtr> reorder_inputs(const dnnl::stream& strm, const std::vector<MemoryPtr>& inMemPtrs);
+#ifdef OPENVINO_ARCH_X86_64
```

Comment:
I have an alternative proposal to avoid `#ifdef`s: maybe we can introduce this code for all platforms? But for non-x86-64 ones we could place one assert which checks that `m_repacking_impl_type == CPURuntimeConfig::RepackingImplType::NONE`.
If we decided to implement this feature for other platforms, almost all the current functional would be reused anyway.

Reply:
Splited into separate executors.
As for `CPURuntimeConfig` - will be implemented after holidays